client/web-sveltekit/src/lib/search/dynamicFilters/CountBadge.svelte

```diff
@@ -32,8 +32,11 @@
 {/if}
 
 <style lang="scss">
-    span.count :global(span) {
-        background-color: var(--secondary-2);
-        color: var(--text-body);
+    span {
+        display: contents;
+        & :global(span) {
```

Comment:
No need for the `&` here, nested selectors are automatically scoped.
```suggestion
        :global(span) {
```